### PR TITLE
* Added new example of OpenStreetMap layer to showcase

### PR DESCRIPTION
**Pull request by twistedpair**
- Added OpenStreetMap layer example to Showcase
- Added "earth" layer to Mars/Moon example

Part of issue #125

### CODE REVIEW

**Comment by branflake2267** [comment]
Great job!

**Comment by branflake2267** [comment]
LGTM go for it. :)

File: gwt-maps-showcase/src/main/java/com/google/gwt/maps/testing/client/maps/ImageMapTypeWidget.java [hunk 1]
```diff
@@ -69,7 +70,8 @@ private void drawMap() {
 
     // draw map
     MapTypeControlOptions controlOpts = MapTypeControlOptions.newInstance();
-    controlOpts.setMapTypeIds(new String[] { LUNAR_NAME, MARTIAN_NAME });
+    //TODO How it make it label as "Earth" not "Satellite" without redefining that standard map type?
+    controlOpts.setMapTypeIds(new String[] { LUNAR_NAME, MARTIAN_NAME, MapTypeId.SATELLITE.toString() });
```
Comment (branflake2267) [inline gwt-maps-showcase/src/main/java/com/google/gwt/maps/testing/client/maps/ImageMapTypeWidget.java:74]: maybe add an enum value SATELLITE("EARTH") to string could give value()...

Comment (twistedpair) [inline gwt-maps-showcase/src/main/java/com/google/gwt/maps/testing/client/maps/ImageMapTypeWidget.java:74]: True, but "Earth" is not a recognized type in the registry. I guess I'd have to reregister it as "Earth" using the old Satellite type.

**Pull request merged**